darts/models/forecasting/block_rnn_model.py

```diff
@@ -430,11 +449,31 @@ def encode_year(idx):
                     logger=logger,
                 )
 
+        # check we got right activation function specified:
+        if activation is not None:
+            try:
+                getattr(nn, activation)
+            except AttributeError:
+                raise_log(
+                    ValueError(
+                        f"Invalid activation function: {activation}. "
+                        "Please use a valid torch.nn activation function name."
+                    ),
+                    logger=logger,
+                )
+            raise_if(
+                activation and (hidden_fc_sizes is None or len(hidden_fc_sizes) == 0),
+                "The activation function has been set, but the model contains only one linear layer. "
+                "Activation function will not be applied.",
+                logger=logger,
+            )
+
```

Comment:
As suggested by @eschibli, I would add a check that raise an error when activation is None and `hidden_fc_sizes` is greater than 0, to tell the user to select an activation function (otherwise, `hidden_fc_sizes > 1` is useless as you correctly reported).

Reply:
Done